secure-session/src/main/java/com/solace/samples/cloudfoundry/securesession/controller/SolaceController.java

```diff
@@ -154,16 +154,20 @@ public void init() {
         logger.info("Solace client initializing and using SolaceMessagingInfo: " + solaceMessagingServiceInfo);
 
         final JCSMPProperties properties = new JCSMPProperties();
-        properties.setProperty(JCSMPProperties.HOST, solaceMessagingServiceInfo.getSmfTlsHost());
+        properties.setProperty(JCSMPProperties.HOST, solaceMessagingServiceInfo.getSmfHost());
         properties.setProperty(JCSMPProperties.VPN_NAME, solaceMessagingServiceInfo.getMsgVpnName());
         properties.setProperty(JCSMPProperties.USERNAME, solaceMessagingServiceInfo.getClientUsername());
         properties.setProperty(JCSMPProperties.PASSWORD, solaceMessagingServiceInfo.getClientPassword());
-
+        
         properties.setProperty(JCSMPProperties.SSL_VALIDATE_CERTIFICATE, true);
         properties.setProperty(JCSMPProperties.SSL_VALIDATE_CERTIFICATE_DATE, true);
         properties.setProperty(JCSMPProperties.SSL_TRUST_STORE, TRUST_STORE);
         properties.setProperty(JCSMPProperties.SSL_TRUST_STORE_PASSWORD, TRUST_STORE_PASSWORD);
 
+		// If using High Availability, uncomment these lines:
+//		JCSMPChannelProperties channelProperties = (JCSMPChannelProperties) properties
+//	            .getProperty(JCSMPProperties.CLIENT_CHANNEL_PROPERTIES);
+//		channelProperties.setReconnectRetries(-1);
         try {
```

Comment:
Please fix so this follows best practices for reconnecting in an HA environment.